src/app/pages/dashboard/components/profile-header/profile-header.html

```diff
@@ -6,14 +6,14 @@
       </ion-col>
       <ion-col>
         <ion-row>
-          <h2 class="name">{{name}}</h2>
+          <h2 id="driving-examiners-name" class="name">{{name}}</h2>
         </ion-row>
         <ion-row>
           <ion-col class="role">
             <span>{{role}}</span>
           </ion-col>
           <ion-col class="staff-number">
-            <span>Staff number: </span><span class="employee-id">{{employeeId}}</span>
+            <span>Staff number: </span><span id="staff-number" class="employee-id">{{employeeId}}</span>
```

Comment:
i think you've either attached the id to the wrong span, or you've written the wrong id.

you've attached an id of "staff-number" to a span containing the employee id (i know they're technically the same thing but id's should be written exactly as what their depicting)

Reply:
In a cucumber test you want to grab the id which has the element. I dont understand the scope of the test which grabs some other div (in this case another spam)